src/components/SnapshotDetails/__tests__/SnapshotDetailsView.spec.tsx

```diff
@@ -118,4 +131,15 @@ describe('SnapshotDetailsView', () => {
     renderWithQueryClientAndRouter(<SnapshotDetails />);
     screen.queryByText('scn 2');
   });
+
+  it('should render without crashing when snapshot has no labels', () => {
+    watchResourceMock.mockImplementation(getSnapshotWithNoLabels);
+    useParamsMock.mockReturnValue({
+      snapshotName: 'my-test-output-no-labels',
+      applicationName: 'my-test-output',
+    });
+    renderWithQueryClientAndRouter(<SnapshotDetails />);
+    expect(screen.getByTestId('snapshot-header-details')).toBeInTheDocument();
+    expect(screen.getByTestId('snapshot-name').innerHTML).toBe('my-test-output-no-labels');
+  });
```

Comment:
I tested the tests locally and it looks all good :)

![Screenshot 2025-04-10 at 10 04 26](https://github.com/user-attachments/assets/e825d157-73f4-4357-b90c-3ebb717e5485)